curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804dc.md

```diff
@@ -135,7 +135,7 @@ assert(
         <h2>Cat Form</h2>
         <form action="https://freecatphotoapp.com/submit-cat-photo">
 --fcc-editable-region--
-
+          
```

Comment:
The addition of the whitespace in the new line here seems inconsistent with the other changes.

Reply:
Is there a way to quick format the extra spaces?